ui/src/app/pages/admin/admin-users.tsx

```diff
@@ -207,7 +207,7 @@ export const AdminUsers = withUserProfile()(class extends React.Component<Props,
 
   render() {
     const {contentLoaded, filter, loading, users} = this.state;
-    const {enableRasLoginGovLinking} = serverConfigStore.get().config;
+    const {enableBetaAccess, enableComplianceTraining, enableEraCommons, enableDataUseAgreement, enableRasLoginGovLinking} = serverConfigStore.get().config;
```

Comment:
I know admin page is missing that config, but I am not sure if we need hide them.